docs/user-guide.md

```diff
@@ -276,11 +278,13 @@ metadata:
 spec:
   installPlanApproval: Automatic
   name: elasticsearch-operator
-  source: redhat-operator-index
+  source: cs-redhat-operator-index
```

Comment:
why is this prefix being added?

Reply:
It's actually a different issue. Moved to another PR: https://github.com/openshift/appliance/pull/160